dining_guide.rst

```diff
@@ -176,6 +176,30 @@ Bargain Eats
 The University is accessible from the I-5, at the 45th St exit. There is some
 parking on campus [to be confirmed]. Prices / availability TBD.
 
+**Fremont**
+
+*25-30 minutes via public transit*
+
+Fremont is something of a nexus of tech companies.  Google, Tableau, Getty Images, Olis Robotics, and adobe (among others) are there.
```

Comment:
```suggestion
Fremont is something of a nexus of tech companies.  Google, Tableau, Getty Images, Olis Robotics, and Adobe (among others) are there.
```

Reply:
Mariatta, I think I have removed the sentence you didn't like.  Let me know if there are any further changes you think are appropriate, please.

Thank you.

Reply:
Hi Jeff, I think Mariatta was just suggesting you capitalize the "A" in Adobe, the description is fine otherwise. There's also a stray URL for Schilling Cider on line 149, referenced above that probably shouldn't be there.